src/status_im/ui/screens/communities/communities_list_redesign.cljs

```diff
@@ -112,7 +112,7 @@
 (defn other-communities [communities sort-list-by]
   (let [sorted-communities (sort-by sort-list-by communities)]
     [list/flat-list
-     {:key-fn                            community-list-key-fn
+     {:key-fn                            str
```

Comment:
Quick question: is _community_ some kind of entity stored in the database? If so, does it have a primary key, for example `:id`? If it has an ID, it'd be preferable to use `{:key-fn :id`}, as it's much faster. `str` can be quite expensive (see https://cljs.github.io/api/cljs.core/str).